src/fortran/cantera_funcs.f90

```diff
@@ -86,6 +86,12 @@ subroutine ctfunc_getCanteraError(buf)
       ierr = ctgetCanteraError(buf)
     end subroutine ctfunc_getCanteraError
 
+    subroutine ctfunc_turnOffExitOnError(self)
+      implicit none
+      type(phase_t), intent(inout) :: self
+      self%err = ctturnOffExitOnError(self%thermo_id)
+    end subroutine ctfunc_turnOffExitOnError
```

Comment:
I'm a little confused by the interface here - the logger is a global, so why does there need to be an argument to either `ctfunc_turnOffExitOnError` or `ctturnOffExitOnError`? And the value passed to `ctturnOffExitOnError` doesn't seem to match it's argument name ("buflen").